src/main/cpp/t02_dist.cpp

```diff
@@ -22,8 +22,12 @@
 using namespace std;
 
 //function distance
-
+double distance(int x1, int y1, int x2, int y2) {
+	double distance = sqrt(pow((x2 - x1), 2) + pow((y2 - y1), 2));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'distance' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594246)